newsplease/config/config.cfg

```diff
@@ -37,6 +37,9 @@ fallbacks = {
     "Download": None
     }
 
+# Check that the crawler will analyze at least one URL during parse step
```

Comment:
could you please add here that in case the check fails, that the next crawler in the pipeline will be used?
also, i'm wondering regarding the term "should" - does the check not always happen? if it indeed does, could you just rename to check_crawler_has_urls_to_scan ?

Reply:
You're right, "should" is useless here, the check always happens.
Thanks for your feedback 🙏 , I've made the change